core/recipe-dsl-kotlin/src/main/kotlin/com/ing/baker/recipe/kotlindsl/KotlinDsl.kt

```diff
@@ -129,13 +131,14 @@ class RecipeBuilder(private val name: String) {
         noinline function: (T1, T2, T3) -> R
     ) {
         val parameters = function.reflect()?.parameters ?: error("Cannot read parameters")
-        val ingredients = listOf(javaTypeOf<T1>(), javaTypeOf<T2>(), javaTypeOf<T3>())
+        val javaTypes = listOf(javaTypeOf<T1>(), javaTypeOf<T2>(), javaTypeOf<T3>())
```

Comment:
Again we can already fix the javadoc like: sieve [T1, T2, T3, R]

Reply:
Good spot.